conda/e3sm_diags_env_dev.yml

```diff
@@ -4,38 +4,38 @@ channels:
   - defaults
 dependencies:
   # Base
+  - python>=3.8, <3.9
```

Comment:
After updating the packages to the latest versions, there are still conflicts in the environment with `python>= 3.9.

Pinning to `python>=3.8, <3.9` instead.

Reply:
@tomvothecoder, I believe this is the reason you weren't able to update to python 3.9: https://github.com/conda-forge/flake8-isort-feedstock/pull/9

Reply:
Thanks @xylar, I think I ran into this when I tried using `python>=3.9` and `flake8-isort` on xCDAT.
Glad to see you opened up a PR for it.

Reply:
If that PR gets merged soon, we can give Python 3.9 a go again, either in `rc3` or the next release. At least Python 3.8 is working without issue.